src/adapters/collateralized_simple_interest_loan_adapter.ts

```diff
@@ -296,6 +313,21 @@ export class CollateralizedSimpleInterestLoanAdapter implements Adapter.Interfac
         ).toArray();
     }
 
+    /**
+     * Seizes the collateral from the given debt agreement and
+     * transfers it to the debt agreement's current beneficiary
+     * (i.e. the person who "owns" the debt).
+     *
+     * @param agreementId The debt agreement's ID
+     */
+    public async seizeCollateral(agreementId: string) {
+        await this.assertCollateralSeizeable(agreementId);
+
+        const collateralizerContract = await this.contractsAPI.loadCollateralizerAsync();
+
+        collateralizerContract.seizeCollateral.sendTransactionAsync(agreementId);
```

Comment:
You'll want to return the `txHash` here, so that the user can use `awaitTransactionMinedAsync` with it.